pkgs/games/chiaki/default.nix

```
@@ -1,23 +1,27 @@
-{ lib, mkDerivation, fetchFromGitHub
-, cmake, ffmpeg, libopus, qtbase, qtmultimedia, qtsvg, pkgconfig, protobuf
-, python3Packages, SDL2 }:
+{ lib, mkDerivation, fetchgit
+, cmake, ffmpeg, libevdev ? null, libopus, udev ? null, qtbase, qtmultimedia
+, qtsvg , pkgconfig, protobuf , python3Packages, SDL2, stdenv }:
 
+assert stdenv.hostPlatform.isLinux -> (udev != null && libevdev != null);
+
+with stdenv.lib;
 mkDerivation rec {
   pname = "chiaki";
-  version = "1.3.0";
+  version = "2.0.1";
 
-  src = fetchFromGitHub {
+  src = fetchgit {
+    url = "https://git.sr.ht/~thestr4ng3r/chiaki";
     rev = "v${version}";
-    owner = "thestr4ng3r";
-    repo = "chiaki";
     fetchSubmodules = true;
-    sha256 = "07w7srxxr8zjp91p5n1sqf4j8lljfrm78lz1m15s2nzlm579015h";
+    sha256 = "0l532i9j6wmzbxqx7fg69kgfd1zy1r1wlw6f756vpxpgswivi892";
   };
 
   nativeBuildInputs = [
     cmake pkgconfig protobuf python3Packages.python python3Packages.protobuf
   ];
-  buildInputs = [ ffmpeg libopus qtbase qtmultimedia qtsvg protobuf SDL2 ];
+  buildInputs = [ ffmpeg libopus qtbase qtmultimedia qtsvg protobuf SDL2 ]
+                ++ optionals stdenv.hostPlatform.isLinux [ libevdev udev]
+                ++ optional (stdenv.isDarwin) qtmacextras;
```

You need to add `qtmacextras` at the top of the file
+
small consistency suggestion:

```suggestion
                ++ optionals stdenv.isLinux [ libevdev udev ]
                ++ optionals stdenv.isDarwin [ qtmacextras ];
```

---

done.